tests/vehicle_power_test.cpp

```diff
@@ -72,7 +72,6 @@ TEST_CASE( "vehicle power with reactor and solar panels", "[vehicle][power]" )
             calendar::turn = calendar::turn_zero + calendar::season_length() + 1_days;
             const time_point start_time = sunrise( calendar::turn ) + 3_hours;
             veh_ptr->update_time( start_time );
-            get_weather().weather_override = WEATHER_SUNNY;
```

Comment:
Please tell me that you didn't break this test by allowing the weather to be something other than consistently sunny.

Reply:
I wish I could.